[2025-03-29T09:00:06.535+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-29T09:00:06.583+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality_monitoring_v3.initialize_database scheduled__2025-03-29T08:40:00+00:00 [queued]>
[2025-03-29T09:00:06.596+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality_monitoring_v3.initialize_database scheduled__2025-03-29T08:40:00+00:00 [queued]>
[2025-03-29T09:00:06.596+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-29T09:00:06.612+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): initialize_database> on 2025-03-29 08:40:00+00:00
[2025-03-29T09:00:06.630+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'air_quality_monitoring_v3', 'initialize_database', 'scheduled__2025-03-29T08:40:00+00:00', '--job-id', '575', '--raw', '--subdir', 'DAGS_FOLDER/data_enriched_predict.py', '--cfg-path', '/tmp/tmpfsgdgh8p']
[2025-03-29T09:00:06.633+0000] {standard_task_runner.py:105} INFO - Job 575: Subtask initialize_database
[2025-03-29T09:00:06.633+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=39180) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-29T09:00:06.634+0000] {standard_task_runner.py:72} INFO - Started process 39205 to run task
[2025-03-29T09:00:06.789+0000] {task_command.py:467} INFO - Running <TaskInstance: air_quality_monitoring_v3.initialize_database scheduled__2025-03-29T08:40:00+00:00 [running]> on host 04906466de51
[2025-03-29T09:00:07.092+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='air_quality_monitoring_v3' AIRFLOW_CTX_TASK_ID='initialize_database' AIRFLOW_CTX_EXECUTION_DATE='2025-03-29T08:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-29T08:40:00+00:00'
[2025-03-29T09:00:07.094+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-29T09:00:07.171+0000] {base.py:84} INFO - Retrieving connection 'postgres_data'
[2025-03-29T09:00:07.336+0000] {data_enriched_predict.py:73} INFO - Tables créées/réinitialisées avec succès
[2025-03-29T09:00:07.365+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-29T09:00:07.406+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-29T09:00:07.407+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=air_quality_monitoring_v3, task_id=initialize_database, run_id=scheduled__2025-03-29T08:40:00+00:00, execution_date=20250329T084000, start_date=20250329T090006, end_date=20250329T090007
[2025-03-29T09:00:07.453+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-29T09:00:07.512+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-29T09:00:07.515+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
